docs/programming_libE.rst

```diff
@@ -4,23 +4,29 @@ Constructing Workflows
 We now give greater detail in programming with libEnsemble.
 
 .. toctree::
+   :maxdepth: 2
+   :caption: The Basics
+
    libe_module
    data_structures/data_structures
    history_output_logging
 
 .. toctree::
-   :caption: libEnsemble User Functions:
+   :caption: Writing User Functions:
+   :maxdepth: 2
 
    function_guides/function_guide_index
 
 .. toctree::
-   :caption: History Array
-
-   function_guides/history_array
+   :caption: Coordinating Apps
+   :maxdepth: 2
 
-.. toctree::
    executor/ex_index
-   resource_manager/resources_index
 
 .. toctree::
+   :maxdepth: 2
+   :caption: Helpful Additions
+
+   function_guides/history_array
```

Comment:
I still think of history array as a central feature of libE, not a helpful addition.

Reply:
I agree

Reply:
👍